relation openstack-dashboard keystone is now ambigious

Traceback (most recent call last):
  File "/usr/bin/juju-deployer", line 9, in <module>
    load_entry_point('juju-deployer==0.10.0', 'console_scripts', 'juju-deployer')()
  File "/usr/lib/python3/dist-packages/deployer/cli.py", line 140, in main
    run()
  File "/usr/lib/python3/dist-packages/deployer/cli.py", line 250, in run
    importer.Importer(env, deployment, options).run()
  File "/usr/lib/python3/dist-packages/deployer/action/importer.py", line 317, in run
    rels_created = self.add_relations()
  File "/usr/lib/python3/dist-packages/deployer/action/importer.py", line 230, in add_relations
    self.env.add_relation(end_a, end_b)
  File "/usr/lib/python3/dist-packages/deployer/env/go.py", line 70, in add_relation
    return self.client.add_relation(endpoint_a, endpoint_b)
  File "/usr/lib/python3/dist-packages/jujuclient/environment.py", line 377, in add_relation
    return self.service.add_relation(*args, **kws)
  File "/usr/lib/python3/dist-packages/jujuclient/juju2/facades.py", line 1053, in add_relation
    'endpoints': [endpoint_a, endpoint_b]
  File "/usr/lib/python3/dist-packages/jujuclient/facades.py", line 72, in rpc
    return self.env._rpc(self.check_op(op))
  File "/usr/lib/python3/dist-packages/jujuclient/rpc.py", line 42, in _rpc
    raise EnvError(result)
jujuclient.exc.EnvError: <Env Error - Details:
 {   'error': 'ambiguous relation: "openstack-dashboard keystone" could refer '
             'to "keystone:websso-trusted-dashboard '
             'openstack-dashboard:websso-trusted-dashboard"; '
             '"openstack-dashboard:identity-service keystone:identity-service"',
    'request-id': 25,
    'response': {}}
 >
2018-05-16 08:07:52 [ERROR] Juju Status: model:
  name: default
  type: iaas
  controller: serverstack-serverstack
  cloud: serverstack
  region: serverstack
  version: 2.4-rc1
  model-status:
    current: available
    since: 16 May 2018 07:47:01Z
  sla: unsupported

--- a/helper/bundles/baremetal7-next.yaml
+++ b/helper/bundles/baremetal7-next.yaml
@@ -159,7 +159,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/baremetal7.yaml
+++ b/helper/bundles/baremetal7.yaml
@@ -159,7 +159,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/designate-next-ha.yaml
+++ b/helper/bundles/designate-next-ha.yaml
@@ -144,7 +144,7 @@ base-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/dynamic-routing-next.yaml
+++ b/helper/bundles/dynamic-routing-next.yaml
@@ -136,7 +136,7 @@ base-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/full-dvr-next.yaml
+++ b/helper/bundles/full-dvr-next.yaml
@@ -101,7 +101,7 @@ openstack-services:
     - [ cinder, nova-cloud-controller ]
     - [ cinder, keystone ]
     - [ cinder, ceph-mon ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/full-next.yaml
+++ b/helper/bundles/full-next.yaml
@@ -110,7 +110,7 @@ openstack-services:
     - [ cinder, keystone ]
     - [ cinder, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/full-ssl-next.yaml
+++ b/helper/bundles/full-ssl-next.yaml
@@ -114,7 +114,7 @@ openstack-services:
     - [ cinder, keystone ]
     - [ cinder, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/full-ssl.yaml
+++ b/helper/bundles/full-ssl.yaml
@@ -114,7 +114,7 @@ openstack-services:
     - [ cinder, keystone ]
     - [ cinder, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/full-vrrpha-next.yaml
+++ b/helper/bundles/full-vrrpha-next.yaml
@@ -66,7 +66,7 @@ openstack-services:
     - [ cinder, nova-cloud-controller ]
     - [ cinder, keystone ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
 openstack-singlerabbit:
   inherits: openstack-services
   relations:

--- a/helper/bundles/full.yaml
+++ b/helper/bundles/full.yaml
@@ -116,7 +116,7 @@ openstack-services:
     - [ cinder, keystone ]
     - [ cinder, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/ha-next.yaml
+++ b/helper/bundles/ha-next.yaml
@@ -184,7 +184,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/ha.yaml
+++ b/helper/bundles/ha.yaml
@@ -187,7 +187,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/haphase2-next.yaml
+++ b/helper/bundles/haphase2-next.yaml
@@ -137,7 +137,7 @@ openstack-services:
     - [ cinder-ceph, ceph ]
     - [ neutron-gateway, nova-cloud-controller ]
     - [ "neutron-gateway:amqp", rabbitmq-server ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ openstack-dashboard, dashboard-hacluster ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-proxy-hacluster ]

--- a/helper/bundles/haphase2.yaml
+++ b/helper/bundles/haphase2.yaml
@@ -141,7 +141,7 @@ openstack-services:
     - [ neutron-gateway, nova-cloud-controller ]
     - [ neutron-gateway, neutron-api ]
     - [ "neutron-gateway:amqp", rabbitmq-server ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ openstack-dashboard, dashboard-hacluster ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-proxy-hacluster ]

--- a/helper/bundles/ksv3-full-next.yaml
+++ b/helper/bundles/ksv3-full-next.yaml
@@ -119,7 +119,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

--- a/helper/bundles/ksv3-full.yaml
+++ b/helper/bundles/ksv3-full.yaml
@@ -119,7 +119,7 @@ openstack-services:
     - [ cinder, cinder-ceph ]
     - [ cinder-ceph, ceph-mon ]
     - [ neutron-gateway, nova-cloud-controller ]
-    - [ openstack-dashboard, keystone ]
+    - [ "openstack-dashboard:identity-service", keystone ]
     - [ swift-proxy, keystone ]
     - [ swift-proxy, swift-storage-z1 ]
     - [ swift-proxy, swift-storage-z2 ]

